src/app/home-page/widget/entries/entries.component.ts

```diff
@@ -0,0 +1,56 @@
+import { Component, OnInit } from '@angular/core';
+import { UserQuery } from 'app/shared/user/user.query';
+import { Observable, Subject } from 'rxjs';
+import { DefaultService } from 'app/shared/openapi';
+import { EntryUpdateTime } from 'app/shared/openapi/model/entryUpdateTime';
+import { debounceTime, takeUntil } from 'rxjs/operators';
+import { formInputDebounceTime } from 'app/shared/constants';
+
+@Component({
+  selector: 'app-entries',
+  templateUrl: './entries.component.html',
+  styleUrls: ['./entries.component.scss']
+})
+export class EntriesComponent implements OnInit {
+  public myEntries;
+  public entryTypeEnum = EntryUpdateTime.EntryTypeEnum;
+  public entryFilterText;
+  public hasEntries = false;
+  public firstCall = true;
+  userId$: Observable<number>;
+  protected ngUnsubscribe: Subject<{}> = new Subject();
+
+  constructor(private userQuery: UserQuery, private defaultService: DefaultService) {}
```

Comment:
I thought you changed the name of DefaultService with an annotation on the webservice side? Is the package.json pointing to an older web service build?

Reply:
I haven't changed the name. I think it was assigned to someone else to do.